index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <!-- <link rel="stylesheet" type="text/css" href="dist/styles.css"> -->
+        <title>Smart Ring</title>
+    </head>
+    <body>
+        <div class="header">
+            <a class="logo" href="#">
+                <span class="logo__text logo__text_color_black">E</span><span class="logo__text logo__text_color_red">market</span><span class="logo__text logo__text_color_black">.</span>
+            </a>
+            <div class="menu">
+                <a class="menu__item menu__item_selected" href="#">home</a>
+                <a class="menu__item" href="#">products</a>
+                <a class="menu__item" href="#"><span class="menu__item_search_icon"></span>search</a>
+                <a class="menu__item" href="#"><span class="menu__item_user_icon"></span>Stive Hense</a>
+            </div>
+        </div>
+        <div class="panel panel_left">
+            <a class="panel__item panel__item_menu_icon" href="#"></a>
+            <a class="panel__item panel__item_more_icon" href="#"></a>
+            <a class="panel__item panel__item_box_icon" href="#"></a>
+        </div>
+        <div class="section">
+            <img class="section__image" src="assets/images/ring.jpg">
+            <div class="product section__product">
+                <div class="product__category">Technology</div>
+                <h1 class="product__name">Smart Ring</h1>
+                <p class="product__description">Morbi in sem quis dui placerat ornare. Pellentesque odio nisi, euismod in, pharetra a, ultrisiet indiam</p>
+                <img class="product__video" src="assets/images/video.jpg">
+                <div class="purchase product__purchase">
+                    <h1 class="purchase__price">199 usd</h1>
+                    <a class="purchase__button" href="#">add to bag</a>
+                </div>
+                </div>
```

Comment:
hm, is this `</div>` miss his `<div>` brother?

Reply:
Oops...